docs/deployment/docker-storage-and-tmp.md

```
@@ -0,0 +1,201 @@
+Active Storage's `blob.open` writes temp files to `/tmp` via Ruby's `Tempfile.open`. This is where the large attack resource downloads land during ingest jobs.
+
+### /rails/tmp — Rails application temp directory
+
+Rails uses `/rails/tmp` for Bootsnap bytecode cache (`tmp/cache/bootsnap/`), Puma PID files (`tmp/pids/`), and other framework temp files. While individual files are small, they accumulate over the lifetime of a container and can exhaust the overlay filesystem on long-running deployments.
+
+### Configuration
+
+```yaml
+# Both web and sidekiq services
+tmpfs:
+  - /tmp:size=${TMPFS_TMP_SIZE:-512m},mode=1777
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)